ramls/eholdings.raml

```diff
@@ -16,7 +16,7 @@ traits:
         type: string
         description: Text entered in search field
         example: Basket Weaving
-        required: true
+        required: false
```

Comment:
Looks good -- just one small comment --think this needs to be applied to indexable trait -- since it should apply to packages/providers and titles (n=both required and default)